src/codemap_details/Components/CodeMap.js

```diff
@@ -1,12 +1,17 @@
 import React, { Component } from "react";
 import CodeMapCard from "./CodeMapCard";
+import styles from "../Css/CodeMap.module.css";
```

Comment:
@Manvityagi Do we have a folder named `Css`? If yes, please change it to `CSS`.

Reply:
@Manvityagi Updated the imports to ../CSS/ 
Also, updated assets to Assets while importing in homepage folder, likewise the rest.  
Please change both the folders.

Reply:
@harsha14gupta Change the folder names in your PR.